src/Admin/Settings/AccessControlSettings.php

```diff
@@ -63,6 +63,16 @@ public static function get_config(): array {
 					'required'    => true,
 					'type'        => 'boolean',
 				],
+				'hasLogoutMutation'                => [
+					'advanced'    => true,
+					'default'     => false,
+					'description' => __( 'Enable Logout Mutation', 'wp-graphql-headless-login' ),
+					'help'        => __( 'Enable this option to allow logout mutations in WP GraphQL. This is useful when using cookie authentication, as it allows logout session requests. Note: "Set authentication cookie" and `Access-Control-Allow-Credentials` must be enabled.', 'wp-graphql-headless-login' ),
```

Comment:
> Note: "Set authentication cookie" and `Access-Control-Allow-Credentials` must be enabled.

This is the part that concerns me, also around the handling, but primarily around the UX.

I think the correct approach is to separate the cookie controls from the access controls both structurally and visually.

That said, this doesn't necessarily need to be handled to merge this and fix #104, so long as we give some forethought to the data-storage so it's future-compatible.

Reply:
Yes I also was thinking about this visually it should be more cleaner and seperated so its easier to understand. Once you enable the cookie authentication you can change the cookie settings. Maybe in a seperate PR would be better.